scout/adapter/mongo/variant.py

```diff
@@ -671,7 +671,7 @@ def delete_variants(self, case_id, variant_type, category=None):
         result = self.variant_collection.delete_many(query)
         LOG.info("{0} variants deleted".format(result.deleted_count))
 
-    def overlapping(self, variant_obj):
+    def overlapping(self, variant_obj, limit=30):
```

Comment:
Won't this slow down the loading of the variantS page?

Reply:
I do not see how it could. We move the limit constant (30) from below up to a named argument, defaulted to the same constant.

Reply:
Ah, or do you mean you would like to also increase the limit to 100 when we are at it? We do have that old heuristic of 30 at many places. It would be a bit interesting to replace them with a system wide cutoff constant or two, and titrate it. This is not the place to change it. We could investigate changing it in a separate PR!

Reply:
Cheers; I also made an issue #4543 so we don't totally forget. While I don't know that we have a reason to change it (and some of it is for loading, so stored on existing cases already), but it is nice to collect and be explicit about constants in code.